index.js

```diff
@@ -86,12 +102,35 @@ function serveStatic (root, options) {
     var forwardError = !fallthrough
     var originalUrl = parseUrl.original(req)
     var path = parseUrl(req).pathname
+    var fullpath, realpath
 
     // make sure redirect occurs at mount
     if (path === '/' && originalUrl.pathname.substr(-1) !== '/') {
       path = ''
     }
 
+    if (followsymlinks === false) {
+      fullpath = realroot + path
+      try {
+        realpath = fs.realpathSync(realroot + path)
```

Comment:
Since this is happening in the even loop of a request, we should probably use the async of this, not sync so it doesn't block unnecessarily